### ISSUE TITLE
Removing dependency on jquery

### ISSUE THREAD

**Issue by josh7weaver** [Shopify/theme-scripts#102]
Hi! I wasn't sure where to ask - but would the maintainers be open to a PR removing Jquery as a dependency? Didn't want to spend time on it if you all are happy with what you have.

Thanks!

**Comment by t-kelly** [comment]
Yes please! It's something we've been slowly moving the libraries towards. Any help would be appreciated.
